client/my-sites/preview/style.scss

```
@@ -1,4 +1,16 @@
 .main.preview {
 	height: 100%;
 	max-width: none;
+
+	.web-preview__toolbar {
+		position:sticky;
+		z-index:1;
```

Nitpick: Ideally this should follow our CSS coding standards.
```suggestion
		position: sticky;
		z-index: 1;
```